bin/node/runtime/src/lib.rs

```diff
@@ -643,7 +641,7 @@ construct_runtime!(
 		EthBacking: pallet_eth_backing::{Module, Call, Storage, Config<T>, Event<T>},
 		EthRelay: pallet_eth_relay::{Module, Call, Storage, Config<T>, Event<T>},
 		Kton: pallet_kton::{Module, Call, Storage, Config<T>, Event<T>},
-		Balances: pallet_ring::{Module, Call, Storage, Config<T>, Event<T>},
+		Ring: pallet_ring::{Module, Call, Storage, Config<T>, Event<T>},
```

Comment:
We should keep to use Balances as Module name, for easier compatibility with polkadot.js apps,

Refer https://github.com/darwinia-network/darwinia/pull/311

Reply:
```Ring``` is our default Balances, so should still use ```Balances``` here.

Reply:
Fixed.